index.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
-var Requested = require('./requested')
-  , https = require('https')
-  , http = require('http');
+module.exports = (function () {
+  if ('XMLHttpRequest' in global) {
```

Comment:
There doesn't need to be any browser testing in this file as the `browser` property of the `package.json` should just point to the `browser.js` file.